frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/color_picker.dart

```diff
@@ -54,7 +54,7 @@ class FlowyColorPicker extends StatelessWidget {
   ) {
     Widget? checkmark;
     if (selected == option.color) {
-      checkmark = svgWidget("grid/checkmark");
+      checkmark = const FlowySvg(FlowySvgData("grid/checkmark"));
```

Comment:
svgs in `flowy_infra_ui` don't know about the generated svgs because the generated svgs go into `lib/generated` so we have to use the raw asset path unfortunately.